contribs/application/src/main/java/org/matsim/application/prepare/freight/GenerateGermanWideFreightTrips.java

```diff
@@ -163,6 +168,19 @@ public Integer call() throws Exception {
 
         MutableInt totalGeneratedPerson = new MutableInt();
 
+        //analysis cross broader trips
+        boolean includeInternationalTrips = false;
+        List<Id<Link>> boundaryLinkIds = new ArrayList<>();
+        if (pathToBoundaryLinks != null) {
+            includeInternationalTrips = true;
+            BufferedReader csvReader = new BufferedReader(new FileReader(pathToBoundaryLinks.toString()));
```

Comment:
The reader is not closed and also uses old style syntax. Use try-with-resource block and nio api.
```try (BufferedReader csvReader = Files.newBufferedReader(pathToBoundaryLinks)) {...}```